### DISCUSSION TITLE
BLACKARCH INSTALLATION

### DISCUSSION THREAD

**Discussion by M0C3** [BlackArch/blackarch#3092] [General]
Type: Question
Can i get any assistance from this team please?? i have been lately installing BLACKARCH_SLIM_2021.iso and i experience this error when installing.

**"mounting '/dev/disk/by-label/BLACKARCH_202103' to '/run/archiso/bootmnt'
waiting 30 seconds for device /dev/disk/by-label/BLACKARCH_202103.."

ERROR: '/dev/disk/by-label/BLACKARCH_202103' device did not show up after 30 seconds.. falling back to interactive prompt
you can try to fix the problem manually, log out when you are finished
can't access tty: job control turned off
[Rootfs: ]#**

can i get any solution to this??
the error in the following image:>>
![WhatsApp Image 2021-05-03 at 12 37 38 PM](https://user-images.githubusercontent.com/58890748/116865228-e6b56080-ac11-11eb-81f9-c4bd6e6c5976.jpeg)

**Comment by ikstream** [comment]
For future reference, I am linking the issue you opened here #3091
